src/config/config.rs

```diff
@@ -538,6 +544,13 @@ pub struct Http {
   pub encoding: Encoding,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq, Eq, schemars::JsonSchema)]
+pub struct Call {
+  #[serde(default, skip_serializing_if = "is_default")]
+  pub query: Option<String>,
+  pub args: HashMap<String, String>,
+}
```

Comment:
Add rust docs. This will be used in generating `tailcallrc.schema.json`

Reply:
done